packages/manager/modules/bm-server-components/src/netboot/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,40 @@
+  "server_configuration_netboot_unavailable": "Information indisponible",
+  "server_configuration_netboot_title_help_rootdevice": "Root devices acceptés (N un nombre) :",
+  "server_configuration_netboot_title_help_rootdevice_list1": "/dev/hda[0 - N]",
+  "server_configuration_netboot_title_help_rootdevice_list2": "/dev/hdb[0 - N]",
+  "server_configuration_netboot_title_help_rootdevice_list3": "/dev/sda[0 - N]",
+  "server_configuration_netboot_title_help_rootdevice_list4": "/dev/sdb[0 - N]",
+  "server_configuration_netboot_title_help_rootdevice_list5": "/dev/md[0 - N]",
+  "server_configuration_netboot_title_help_rootdevice_list6": "/dev/rd/c0p0[0 - N]",
+  "server_configuration_netboot_title_help_rootdevice_list7": "/dev/ram0",
+  "server_configuration_netboot_current": "Netboot acutellement configuré sur votre serveur {{t0}} :",
+  "server_configuration_netboot_step1_question": "Que souhaitez-vous faire ?",
+  "server_configuration_netboot_option_harddisk": "Booter sur le disque dur",
+  "server_configuration_netboot_option_rescue": "Booter en mode rescue",
+  "server_configuration_netboot_option_network": "Booter en mode network",
+  "server_configuration_netboot_configuration": "Configuration :",
+  "server_configuration_netboot_rescue_choice": "Rescue disponible :",
+  "server_configuration_netboot_rescue_mail": "Recevoir les identifiants du mode sur l'adresse e-mail :",
+  "server_configuration_netboot_rescue_mail_invalid": "Invalide",
+  "server_configuration_netboot_kernel_choice": "Kernel disponible :",
+  "server_configuration_netboot_kernel_option": "Options prise en charges par ce kernel :",
+  "server_configuration_netboot_kernel_rootdevice": "Root device :",
+  "server_configuration_netboot_kernel_rootdevice_exemple": "exemple: voir aide",
+  "server_configuration_netboot_loading_error": "Une erreur est survenue lors du chargement des informations du netboot sur le serveur {{t0}}",
+  "server_configuration_netboot_step2_question": "Etes-vous certain de vouloir appliquer les changements suivant sur votre serveur {{t0}} ?",
+  "server_configuration_netboot_step2_resume": "Résumé :",
+  "server_configuration_netboot_step2_type": "Type de netboot :",
+  "server_configuration_netboot_step2_kernel": "Nom du kernel :",
+  "server_configuration_netboot_step2_rescue": "Nom du rescue :",
+  "server_configuration_netboot_step2_rescue_mail": "Courriel du rescue :",
+  "server_configuration_netboot_success": "Le netboot a correctement été mis à jour. Vous devez redémarrer le serveur.",
+  "server_configuration_netboot_fail": "Une erreur est survenue lors de la mise à jour du netboot sur le serveur {{t0}}",
+  "server_configuration_netboot_next": "Suivant",
+  "server_configuration_netboot_cancel": "Annuler",
+  "server_configuration_netboot_confirm": "Valider",
+  "server_configuration_netboot_review": "Résumé",
+  "server_configuration_netboot_save_in_progress": "Mise à jour...",
```

Comment:
```suggestion
  "server_configuration_netboot_save_in_progress": "Mise à jour…",
```

Reply:
These are the existing translations. I am not planning to make any changes to them. Its not part of this refactoring scope.